shared/fs/footer/upload-container.js

```diff
@@ -33,16 +34,22 @@ const mapDispatchToProps = (dispatch: Dispatch) => ({
   debugToggleShow: getDebugToggleShow(dispatch),
 })
 
-const mergeProps = ({_uploads}, {debugToggleShow}) =>
-  ({
+const mergeProps = ({_uploads, _pathItems}, {debugToggleShow}) => {
+  const folders = _pathItems.filter(path => path.type === 'folder')
+  const uploadPaths = _uploads.syncingPaths // get array of paths
+  const filePaths = uploadPaths.filter(item => !folders.get(item))
+
+  return ({
     // We just use syncingPaths rather than merging with writingToJournal here
     // since journal status comes a bit slower, and merging the two causes
     // flakes on our perception of overall upload status.
     files: _uploads.syncingPaths.size,
+    filePaths,
```

Comment:
As I mentioned in the previous comment, I think we'd need to turn path into a name here. Also we only need one filename here. WDYT about something like this?

```
fileName: filePaths.size() === 1 ? filePaths.first() : undefined,
```